monj06-sp

mondsjar opened the pull request:
Odevzdávám pilotní verzi semestrální práce

mondsjar committed:
monj06-sp

Odevzdávám pilotní verzi semestrální práce

mondsjar committed:
aktualizace sp

https://eso.vse.cz/~monj06/4IZ278/sp/app/home

mondsjar commented:
admin:
email: root@asdf.com
password: asdfasdf

Bohužel po nahrání na eso některé věci v aplikaci nefungují. Používal jsem při vývoji livewire. Nevím jestli je s tím nějaký problém na eso serveru.

nvbach91 commented:
jeste prosim vas o odkaz do vasi aplikace

mondsjar commented:
Pardon, byl u druhého commitu.
https://eso.vse.cz/~monj06/4IZ278/sp/app/home

nvbach91 commented:
nefunguje to

![image](https://github.com/user-attachments/assets/c4e8fc5c-0924-4f4b-bfb2-328b1ae417cc)

mondsjar commented:
Nevím, mně ten odkaz funguje.

nvbach91 commented:
- nektere dulezite texty vubec nejsou viditelne, proc? [image](https://github.com/user-attachments/assets/8b27f469-a7b1-49cb-8763-642c2dc1f892), [image](https://github.com/user-attachments/assets/dc13f65d-0002-4409-89ec-8dea62201cce)

- kde je stranka profilu uzivatele? jak se uzivatel odhlasi? nejde vubec klikat na tlacitko uzivatele
- pri vyplnovani kontaktnich informaci pro objednavku vubec nevidim co pisu...
- udaje ve formulare se ztrati po neuspesnem pokusu
- nerozumim proc zip field must not be greater than 255 [image](https://github.com/user-attachments/assets/15ef7eb3-b2c2-4836-ae65-d914fa2617ea), [image](https://github.com/user-attachments/assets/56ba25a4-eec0-4c19-ad64-71bb65c27228), 
[image](https://github.com/user-attachments/assets/1d01efa4-04db-4462-9ce3-6b66e457ccdb)
- jak se dostanu do administrace?
- testoval jste to vubec?

mondsjar commented:
> admin: email: [root@asdf.com](mailto:root@asdf.com) password: asdfasdf
> 
> Bohužel po nahrání na eso některé věci v aplikaci nefungují. Používal jsem při vývoji livewire. Nevím jestli je s tím nějaký problém na eso serveru.

Jak jsem psal výše na esu se to trochu rozbilo (např. nefunguje tam dropdown, kde je třeba ta administrace). U sebe jsem to testoval a většina věcí co jste psal funguje.

nvbach91 commented:
- pridat menu k cenam produktu
- kosik - nazev produktu
- nefunguje odecteni skladu
- uzivatel nemuze zobrazit sve objednavky?
- admin musi mit moznost vyrizovat objednavky, aktualizovat stav, filtrovat, ...

mondsjar committed:
aktualizace sp

oprava sp podle instrukcí

mondsjar committed:
aktualizace sp

Oprava zobrazení objednávek v admin order. Nejnovější nahoře a více informací v záhlaví accordionu pro snazší orientaci.

nvbach91 commented:
objednavky stale nejdou zobrazit

![image](https://github.com/user-attachments/assets/d164e4a8-dceb-4081-802c-cadbfa63c17a)

mondsjar commented:
Už by to mělo jít.
